From c94c9df33b2fe43836b872a7c8d1d96f96a47061 Mon Sep 17 00:00:00 2001
From: Ken <21211439+kencu@users.noreply.github.com>
Date: Wed, 1 Jul 2020 02:29:05 -0700
Subject: [PATCH] main.py : don't try libc.prctl on MacOS (#973)

It's not available on macOS, and so generates a warning every launch.
---
 modules/main.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules/main.py b/modules/main.py
index 47664b32..d3f6115b 100644
--- a/modules/main.py
+++ b/modules/main.py
@@ -167,7 +167,7 @@ def initializations():
     if hasattr(sys, 'frozen'):
         import warnings
         warnings.filterwarnings(cons.STR_IGNORE)
-    if not cons.IS_WIN_OS:
+    if not (cons.IS_WIN_OS or cons.IS_MAC_OS):
         try:
             # change process name
             import ctypes, ctypes.util
